src/Symfony/Component/Console/Command/LazyCommand.php

```diff
@@ -0,0 +1,207 @@
+    }
+
+    public function isEnabled(): bool
+    {
+        return $this->isEnabled ?? $this->getCommand()->isEnabled();
+    }
+
+    public function run(InputInterface $input, OutputInterface $output): int
+    {
+        return $this->getCommand()->run($input, $output);
+    }
+
+    /**
+     * @return $this
+     */
+    public function setCode(callable $code): self
+    {
+        $this->getCommand()->setCode($code);
+
+        return $this;
+    }
+
+    /**
+     * @internal
+     */
+    public function mergeApplicationDefinition(bool $mergeArgs = true): void
+    {
+        $this->getCommand()->mergeApplicationDefinition($mergeArgs);
+    }
+
+    /**
+     * @return $this
+     */
+    public function setDefinition($definition): self
+    {
+        $this->getCommand()->setDefinition($definition);
+
+        return $this;
+    }
+
+    public function getDefinition(): InputDefinition
+    {
+        return $this->getCommand()->getDefinition();
+    }
+
+    public function getNativeDefinition(): InputDefinition
+    {
+        return $this->getCommand()->getNativeDefinition();
+    }
+
+    /**
+     * @return $this
+     */
+    public function addArgument(string $name, int $mode = null, string $description = '', $default = null): self
+    {
+        $this->getCommand()->addArgument($name, $mode, $description, $default);
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function addOption(string $name, $shortcut = null, int $mode = null, string $description = '', $default = null): self
+    {
+        $this->getCommand()->addOption($name, $shortcut, $mode, $description, $default);
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function setProcessTitle(string $title): self
+    {
+        $this->getCommand()->setProcessTitle($title);
+
+        return $this;
+    }
+
+    /**
+     * @return $this
+     */
+    public function setHelp(string $help): self
+    {
+        $this->getCommand()->setHelp($help);
+
+        return $this;
+    }
+
+    public function getHelp(): string
+    {
+        return $this->getCommand()->getHelp();
+    }
+
+    public function getProcessedHelp(): string
+    {
+        return $this->getCommand()->getProcessedHelp();
+    }
+
+    public function getSynopsis(bool $short = false): string
+    {
+        return $this->getCommand()->getSynopsis($short);
+    }
+
+    /**
+     * @return $this
+     */
+    public function addUsage(string $usage): self
+    {
+        $this->getCommand()->addUsage($usage);
+
+        return $this;
+    }
+
+    public function getUsages(): array
+    {
+        return $this->getCommand()->getUsages();
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getHelper(string $name)
+    {
+        return $this->getCommand()->getHelper($name);
+    }
+
+    public function getCommand(): parent
+    {
+        if (!$this->command instanceof \Closure) {
```

Comment:
Can this variable be anything but \Closure it is mandatory and not nullable on the constructor?

Reply:
yes, see the line below this one, where the variable is resolved into a `Command`